### DISCUSSION TITLE
Button (colorScheme) Color Question

### DISCUSSION THREAD

**Discussion by akiarostami** [chakra-ui/chakra-ui#2939] [Q&A] [answered]
Hello,

I have a *very* simple task. I want to set the text color of a button text to Black or White. But Buttons only have colorScheme, and accept color objects (e.g. "green", with 100-900 gradient). Black and White are simple color values. The only solution is to use a Text object inside the button and set the text color there, which doesn't make sense to me.

Is there a way to set the color of a Button's text to Black or White? I feel I'm missing something.

Thank you for you help!

**Comment by chr-ge** [comment, marked as answer]
You can set a colorScheme and text color on a button:
```jsx
<Button colorScheme='blue' color='black'>Hello</Button>
<Button colorScheme='yellow' color='white'>World</Button>
```

**Reply by akiarostami** [reply]
huh! I swear I did this and it didn't work! :)
Thank you for your help @chr-ge. I appreciate it.
